src/Paysera/Bundle/ClientReleaseBundle/Service/ReleaseStep/GeneratePhpClientStep.php

```diff
@@ -19,7 +19,7 @@ public function processStep(ReleaseStepData $releaseStepData, InputInterface $in
         $releaseStepData->setGeneratedDir($releaseStepData->getTempDir() . '/generated');
 
         $args = [
-            'raml-code-generator',
+            'bin/raml-code-generator',
```

Comment:
The `raml-code-generator` should be globally installed, not in the bin dir of every project

Reply:
maybe it should try to look for globally installed and local binary.